### ISSUE TITLE
feature: expand all in Tree from current tree location downwards

### ISSUE THREAD

**Issue by born2net** [zalmoxisus/redux-devtools-extension#727]
I find myself many times having to click so many times to expand tree selections as I have a deeply nested store.
It would be awesome if we can have an option to expand all tree item below the current click offset.
see below:

![double](https://user-images.githubusercontent.com/5090936/76431984-945f1380-636f-11ea-842e-46a63eb27e42.png)

**Comment by jwschram** [comment]
I totally agree, however probably a modifier key and click (such as **ctrl+click**) would be more in keeping with other applications.

**Comment by fargusplumdoodle** [comment]
Its been a couple years, is there any posibility this will still happen?
